actors/runtime/src/main/java/cloud/orbit/actors/runtime/ShardedReminderController.java

```diff
@@ -0,0 +1,84 @@
+/*
+ Copyright (C) 2017 Electronic Arts Inc.  All rights reserved.
+
+ Redistribution and use in source and binary forms, with or without
+ modification, are permitted provided that the following conditions
+ are met:
+
+ 1.  Redistributions of source code must retain the above copyright
+     notice, this list of conditions and the following disclaimer.
+ 2.  Redistributions in binary form must reproduce the above copyright
+     notice, this list of conditions and the following disclaimer in the
+     documentation and/or other materials provided with the distribution.
+ 3.  Neither the name of Electronic Arts, Inc. ("EA") nor the names of
+     its contributors may be used to endorse or promote products derived
+     from this software without specific prior written permission.
+
+ THIS SOFTWARE IS PROVIDED BY ELECTRONIC ARTS AND ITS CONTRIBUTORS "AS IS" AND ANY
+ EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+ WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ DISCLAIMED. IN NO EVENT SHALL ELECTRONIC ARTS OR ITS CONTRIBUTORS BE LIABLE FOR ANY
+ DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+ ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF
+ THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package cloud.orbit.actors.runtime;
+
+import cloud.orbit.actors.Actor;
+import cloud.orbit.actors.Remindable;
+import cloud.orbit.actors.annotation.NeverDeactivate;
+import cloud.orbit.concurrent.Task;
+
+import java.util.Date;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+@NeverDeactivate
+public interface ShardedReminderController extends Actor
```

Comment:
The interfaces are the same, except for the `@NoIdentity` annotation on `ReminderController`.  If `ShardedReminderController` extends `ReminderController` I get 

```
[OrbitThread-25] ERROR cloud.orbit.actors.net.DefaultHandlerContext - Error writing: 
java.util.concurrent.CompletionException: cloud.orbit.exception.UncheckedException: Timeout waiting for a server capable of handling: cloud.orbit.actors.runtime.ReminderController
	at java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:292)
	at java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:308)
	at java.util.concurrent.CompletableFuture.uniApply(CompletableFuture.java:593)
	at java.util.concurrent.CompletableFuture.uniApplyStage(CompletableFuture.java:614)
	at java.util.concurrent.CompletableFuture.thenApply(CompletableFuture.java:1983)
	at cloud.orbit.concurrent.Task.thenApply(Task.java:395)
	at cloud.orbit.actors.runtime.Hosting.locateAndActivateActor(Hosting.java:382)
	at cloud.orbit.actors.runtime.Hosting.locateActor(Hosting.java:292)
	at cloud.orbit.actors.runtime.Hosting.writeInvocation(Hosting.java:753)
	at cloud.orbit.actors.runtime.Hosting.write(Hosting.java:720)
	at cloud.orbit.actors.net.DefaultHandlerContext.write(DefaultHandlerContext.java:172)
	at cloud.orbit.actors.runtime.Execution.write(Execution.java:62)
	at cloud.orbit.actors.net.DefaultHandlerContext.write(DefaultHandlerContext.java:172)
	at cloud.orbit.actors.net.HandlerAdapter.write(HandlerAdapter.java:92)
	at cloud.orbit.actors.runtime.DefaultResponseCachingExtension.write(DefaultResponseCachingExtension.java:243)
	at cloud.orbit.actors.net.DefaultHandlerContext.write(DefaultHandlerContext.java:172)
	at cloud.orbit.actors.test.TestInvocationLog.write(TestInvocationLog.java:127)
	at cloud.orbit.actors.net.DefaultHandlerContext.write(DefaultHandlerContext.java:172)
	at cloud.orbit.actors.net.DefaultPipeline.write(DefaultPipeline.java:162)
	at cloud.orbit.actors.Stage.invoke(Stage.java:1128)
	at cloud.orbit.actors.runtime.RemoteReference.invoke(RemoteReference.java:102)
	at cloud.orbit.actors.runtime.ReminderController$Reference.ensureStart(ReminderController$Reference.java)
	at cloud.orbit.actors.Stage.startReminderController(Stage.java:890)
	at cloud.orbit.actors.Stage.lambda$start$10(Stage.java:845)
	at cloud.orbit.concurrent.TaskContext.lambda$wrap$0(TaskContext.java:178)
	at java.util.concurrent.CompletableFuture.uniRun(CompletableFuture.java:705)
	at java.util.concurrent.CompletableFuture$UniRun.tryFire(CompletableFuture.java:687)
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:474)
	at java.util.concurrent.CompletableFuture.complete(CompletableFuture.java:1962)
	at cloud.orbit.concurrent.Task.internalComplete(Task.java:123)
	at cloud.orbit.concurrent.Task.lambda$from$1(Task.java:172)
	at java.util.concurrent.CompletableFuture.uniHandle(CompletableFuture.java:822)
	at java.util.concurrent.CompletableFuture$UniHandle.tryFire(CompletableFuture.java:797)
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:474)
	at java.util.concurrent.CompletableFuture.complete(CompletableFuture.java:1962)
	at cloud.orbit.concurrent.Task.internalComplete(Task.java:123)
	at cloud.orbit.concurrent.Task.lambda$from$1(Task.java:172)
	at java.util.concurrent.CompletableFuture.uniHandle(CompletableFuture.java:822)
	at java.util.concurrent.CompletableFuture$UniHandle.tryFire(CompletableFuture.java:797)
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:474)
	at java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1632)
	at java.util.concurrent.CompletableFuture$AsyncRun.exec(CompletableFuture.java:1618)
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157)
Caused by: cloud.orbit.exception.UncheckedException: Timeout waiting for a server capable of handling: cloud.orbit.actors.runtime.ReminderController
	at cloud.orbit.actors.runtime.Hosting.selectNode(Hosting.java:429)
	at cloud.orbit.actors.runtime.Hosting.locateAndActivateActor(Hosting.java:380)
	... 39 more
```

I can dig further if you'd like and figure out why.